From 776cc68d06787c739d034a25201e0d37c4c2aded Mon Sep 17 00:00:00 2001
From: yachun <yachun@marvell.com>
Date: Mon, 21 Aug 2017 16:14:45 +0800
Subject: [PATCH 2153/2241] dts: a3900: update SPI/NAND mtd partitions

mtd#   device  addr        size
mtd0 - AP NOR         0x0    0x500000
mtd1 - AP NOR    0x500000    0xb00000
mtd2 - CP NAND  0x1000000  0x1f800000
mtd3 - CP NAND 0x20800000  0x1f800000

Change-Id: Ic53556760330c356a3e52d037427eb5ff81da60f
Signed-off-by: yachun <yachun@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/48624
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-3900-db.dtsi | 20 ++++----------------
 1 file changed, 4 insertions(+), 16 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi b/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
index e2e0fa5..0ac9619 100644
--- a/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
@@ -74,15 +74,11 @@
 					spi-max-frequency = <20000000>;
 					partition@0 {
 						label = "boot";
-						reg = <0x0 0x200000>;
+						reg = <0x0 0x500000>;
 					};
-					partition@200000 {
-						label = "Filesystem";
-						reg = <0x200000 0xd00000>;
-					};
-					partition@d00000 {
-						label = "boot_2nd";
-						reg = <0xf00000 0x100000>;
+					partition@500000 {
+						label = "env";
+						reg = <0x500000 0xb00000>;
 					};
 				};
 			};
@@ -104,14 +100,6 @@
 				nand-ecc-strength = <4>;
 				nand-ecc-step-size = <512>;
 				status = "okay";
-				partition@0 {
-					label = "boot";
-					reg = <0 0x500000>;
-				};
-				partition@500000 {
-					label = "env";
-					reg = <0x500000 0x400000>;
-				};
 				partition@1000000 {
 					label = "rootfs";
 					reg = <0x1000000 0x1f800000>;
-- 
2.7.4

